### ISSUE TITLE
ImageMagick's import is syntax-highlit as an import statement

### ISSUE THREAD

**Issue by qwenger** [xonsh/xonsh#3612]
I have ImageMagick with its cli tool `import` installed. When I type in e.g.:

```
import -window root test.png
```

the command runs successfully (a screenshot of my screen is saved to test.png), but the syntax highlighting corresponds to python mode (the "import" has the same color as if I write e.g. "import os", not the same as e.g. "echo 1").

Neither `window` nor `root` nor `test` are defined python variables in my shell.

## xonfig

<details>

```
+------------------+---------------------+
| xonsh            | 0.9.18              |
| Git SHA          | 26475146            |
| Commit Date      | May 5 06:42:28 2020 |
| Python           | 3.8.3               |
| PLY              | 3.11                |
| have readline    | True                |
| prompt toolkit   | 3.0.5               |
| shell type       | prompt_toolkit      |
| pygments         | 2.6.1               |
| on posix         | True                |
| on linux         | True                |
| distro           | arch                |
| on darwin        | False               |
| on windows       | False               |
| on cygwin        | False               |
| on msys2         | False               |
| is superuser     | False               |
| default encoding | utf-8               |
| xonsh encoding   | utf-8               |
| encoding errors  | surrogateescape     |
+------------------+---------------------+
```

</details>

## Expected Behavior
The line should be syntax-highlit following subprocess mode, for example the "import" word should be colored the same as "echo".

## Current Behavior
The line seems to be syntax-highlit following python mode.

**Comment by scopatz** [comment]
Yeah, this is a legitimate issue.  The highlighting goes through a totally separate code path from execution, though. PRs welcome!

**Comment by qwenger** [comment]
Yes I see and that's certainly not an easy thing to change. Note that is affects also e.g. indented blocks, for instance

```
def test():
    echo 1
```

Here the `echo` is formatted as Python.

I think that the execer should somehow be the single source of truth regarding the decision on Python vs. subprocess mode, and the lexer/style should consult it on that matter.

I'll see whether I get some time for a PR. But that's not necessarily in a near future.

**Comment by anki-code** [comment]
In xonsh >= 0.23.0 use `$XONSH_BUILTINS_TO_CMD=True` to enable experimental feature: if plane name is python builtin and has executatble or alias use it as cmd (so simple `id`, `zip`, `import` are working).
